ministryofjustice/correspondence_tool_staff
# Ct 2450 Add movement fields to teams and r003

Amend report to show moved team
This is the preparation work for moving business units between directorate and the reporting on this

## Description

 
## Self-review checklist

 
### Screenshots

 
![Screen Shot 2019-10-29 at 14 40 16](https://user-images.githubusercontent.com/6860654/67779558-b741f880-fa5c-11e9-9f32-55941066beb5.png)

### Related JIRA tickets

 https://dsdmoj.atlassian.net/browse/CT-2450?atlOrigin=eyJpIjoiZDc4ZmU3NTczZmJiNGM4ZGFkMTk1YTQ3ZGZkYTNmYzEiLCJwIjoiaiJ9

### Deployment

Added self-referencing migration for the id of the target (new) team 
 
### Manual testing instructions

Run the FOI/Business Unit report, and check to see if Moved to column is present.
You may need to amend your database such that a deactivated team had the foreign key of a destination Business Unit.
Note that this change does NOT include the functionality to move teams, it merely sets up the data structure ready for it and allows reporting.
